tests/Dockerfile

```diff
@@ -55,6 +55,7 @@ RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpi
     ln -s /cache/images /build/images && \
     ln -s /cache/images /build/virt-builder && \
     ln -s /cache/github /build/github && \
+    ln -s /tmp /build/tmp && \
```

Comment:
```/build``` is set to ```$TEST_DATA``` ... and we start to use ```$TEST_DATA/tmp``` in the cockpit repo. There's one open #6733 and one closed #6701 pull request.